getting-started/templates/systemlink-admin-values.yaml

```diff
@@ -11,3 +11,9 @@ strimzi-kafka-operator:
   watchAnyNamespace: true
   podSecurityContext: 
     runAsNonRoot: true
+  ## Clusters that are not allowing external intenet access MUST set properties below.
```

Comment:
Ugh, does this have to duplicate values from our globals section? Can we use references to remove the need to duplicate configuration?

Do we need documentation work to call out this requirement?

Reply:
Actually, now that I'm thinking about it, any settings that require user review should be marked in the comments like we do in the other values files. And we will need a documentation change because the documentation currently says no changes need to be made to this file.

Reply:
@JD-Robertson This file is referenced in the "Installing Cluster Prerequisites", is that when the user needs to update the values? Feel free to make comments directly in the [review repo](https://ni.visualstudio.com/Users/_git/mhilliar?path=/Reviews/SLE/installing-systemlink-enterprise.html&_a=preview).